CMakeLists.txt

```diff
@@ -94,15 +94,18 @@ endif()
 # Add web application
 # -----------------------------------------------------------------------------
 option(TOMVIZ_DOWNLOAD_WEB  "Enable downloading web application." OFF)
-set(tomviz_html_path "${CMAKE_BINARY_DIR}/web/tomviz.html")
+set(tomviz_html_source_path "${CMAKE_SOURCE_DIR}/web/tomviz.html")
```

Comment:
The path I gave you, is the one that is used when building/generating that file. 
I can probably change it to be the root directory of the web, but conceptually, I'd rather keep it separate.